packages/arb-token-bridge-ui/src/token-bridge-sdk/EthDepositStarter.ts

```diff
@@ -0,0 +1,114 @@
+import { EthBridger } from '@arbitrum/sdk'
+import { BigNumber } from 'ethers'
+import {
+  ApproveNativeCurrencyProps,
+  BridgeTransferStarter,
+  BridgeTransferStarterProps,
+  RequiresNativeCurrencyApprovalProps,
+  TransferEstimateGas,
+  TransferProps,
+  TransferType
+} from './BridgeTransferStarter'
+import { getAddressFromSigner, percentIncrease } from './utils'
+import { depositEthEstimateGas } from '../util/EthDepositUtils'
+import { fetchErc20Allowance } from '../util/TokenUtils'
+
+export class EthDepositStarter extends BridgeTransferStarter {
+  public transferType: TransferType = 'eth_deposit'
+
+  constructor(props: BridgeTransferStarterProps) {
+    super(props)
+  }
```

Comment:
(applies to all) this isn't necessary when we're not doing any additional steps in the constructor, the default behaviour is inherited automatically